ui/src/testing/react-test-helpers.tsx

```diff
@@ -188,7 +188,6 @@ export const getDropdownOption = (
     (di) => di.textContent === optionText
   );
   expect(option).toBeInTheDocument();
-  expect(option).toHaveTextContent(optionText);
```

Comment:
this was a little silly, because it's how we searched for the option in the first place